azurelinuxagent/ga/monitor.py

```diff
@@ -178,8 +179,8 @@ def init_sysinfo(self):
         try:
             ram = self.osutil.get_total_mem()
             processors = self.osutil.get_processor_cores()
-            self.sysinfo.append(TelemetryEventParam("RAM", ram))
-            self.sysinfo.append(TelemetryEventParam("Processors", processors))
+            self.sysinfo.append(TelemetryEventParam("RAM", int(ram)))
+            self.sysinfo.append(TelemetryEventParam("Processors", int(processors)))
```

Comment:
do the conversion in get_processor_cores?

Reply:
Similar to the comment above for `get_total_mem`. Only for schema for an event rather than changing the behavior for all the users of this method.